tf-modules/cumulus-rds-tf/variables.tf

```diff
@@ -46,6 +46,12 @@ variable "db_admin_password" {
   type = string
 }
 
+variable "input_security_group_id" {
+  description = "Optional existing security group ID to use for the RDS cluster"
+  type        = string
+  default     = null
```

Comment:
Nit: Can we add something like this to avoid typos?      A lot of our boilerplate doens't have it because it really only became useful in 1.9, but it'd be nice to add validations to new variables. 

```
validation { 
   condition:  var.input_security_group_id == null || can(regex("^sg-[a-f0-9]+$", var.input_security_group_id))
   error_message: "some error message" 
}